ui/src/main/js/application/auth/SamlForm.js

```diff
@@ -90,6 +97,20 @@ const SamlForm = ({ csrfToken, errorHandler, readonly, displayTest, fileDelete,
                 submitLabel="Save SAML Configuration"
                 testLabel="Test SAML Configuration"
             >
+                <LabeledField label={importBlackDuckSSOConfigLabel} description={importBlackDuckSSOConfigDescription}>
```

Comment:
Lets move both the `<LabeledField>` and `<BlackDuckSSOConfigImportModal>` code blocks to line 149.  Since this really only matters when the user has selected the URL option.